index.html

```diff
@@ -67,17 +67,17 @@ <h2>Pick Your Destination</h2>
       <div class="destination">
```

Comment:
By changing the class names, you might have affected the CSS and caused a bigger bug than you originally intended.
I'd change them back to `btn`